pkg/interface/src/apps/groups/components/lib/contact-sidebar.tsx

```diff
@@ -166,8 +169,20 @@ export class ContactSidebar extends Component<ContactSidebarProps, ContactSideba
           >Channels</Link>
           {shareSheet}
           <h2 className="f9 pt4 pr4 pb2 pl4 gray2 c-default">Members</h2>
-          {contactItems}
-          {groupItems}
+          {this.memberbox.current ? <List
```

Comment:
We see if the parent has been mounted yet first so we can set the height to that